Admin/AbstractAdminDefinition.php

```diff
@@ -40,6 +40,13 @@ public function getExportFormats()
     /**
      * {@inheritdoc}
      */
+    public function configureExportFields(&$fields)
```

Comment:
No type hinting? I'm not comfortable with php references, but I don't have any argument to oppose them, so…